scripts/rv_expression_association/subset_variants.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# pylint: disable=no-value-for-parameter,missing-function-docstring,missing-module-docstring
+
+import click
+import hail as hl
+from cpg_utils.hail_batch import dataset_path, init_batch
+
+
+@click.command()
+@click.option('--input-mt-path', required=True)  # 'mt/v7.mt'
+@click.option('--genes', required=True)  # 'LMNA'
+@click.option(
+    '--output-mt-path', required=True
+)  # 'gs://cpg-tob-wgs-main-analysis/tob_wgs_rv/pseudobulk_rv_association/significant_genes_burden_max.mt'
+def subset_variants(
+    input_mt_path: str,
+    genes: str,
+    output_mt_path: str,
+):
+    init_batch()
+
+    # read ht objects to extract variant names
+    variants = []
+    genes_of_interest = genes.split(' ')
```

Comment:
```suggestion
@click.command()
@click.option('--input-mt-path', required=True)  # 'mt/v7.mt'
@click.option(
    '--output-mt-path', required=True
)  # 'gs://cpg-tob-wgs-main-
analysis/tob_wgs_rv/pseudobulk_rv_association/significant_genes_burden_max.mt'
@click.argument('genes_of_interest', nargs=-1)  # positional argument(s)
def subset_variants(
    input_mt_path: str,
    output_mt_path: str,
    genes_of_interest: list[str],
):
    """
    genes_of_interest as a positional argument, accepting
    one or more genes
    """
    
    init_batch()

    # read ht objects to extract variant names
    variants = []
```

syntax for taking genes as a list of strings instead of one string and splitting manually

Reply:
i dont know i seem to remember we tried this previously and failed?

Reply:
This is the difference between `click.option` and `click.argument` - options can take multiple arguments, but due to how they're parsed they have to be a fixed/known number. 
Arguments are positional, and you can take an arbitrary number of input values - these are collected as a tuple

Reply:
I made a little test script to try this out locally:

```
import click


@click.argument('g', nargs=-1)
@click.command()
def main(g):
    print(g)

if __name__ == '__main__':
    main()
```

> python click_test.py a a a
('a', 'a', 'a')